### ISSUE TITLE
Add operator name to error message

### ISSUE THREAD

**Issue by td5r** [Bitspark/slang#234]
e.g.  an error like
`expected map but got "...."` should also provide the operator name, where this error has occurred.
At best provide that operator name via already existing logger in as JSON string

**Comment by kairichard** [comment]
@td5r I agree that is an issue, however I would postpone fixing in order to put time into other issues.

**Comment by jm9e** [comment]
Thanks for reporting this! Knowing which operator failed makes these errors much easier to investigate.

I prepared a fix in #256 that adds structured `operatorId` and `operatorName` fields to panic reports and builtin operator logs while preserving blueprint context. The tests verify an actual panic and 100 concurrent operator log events without identity leaking between them.

The focused tests pass. The fix is awaiting the required review before it can land on master, so I am keeping this issue open until then. Thank you again for highlighting the missing context!

**Comment by jm9e** [comment]
Thank you for reporting this and helping make runtime errors easier to diagnose. The fix is now merged in #256: errors and panic logs include the originating operator name and UUID, with regression coverage for concurrent operators. All GitHub Actions checks passed.
